src/components/profile/ProfileService.js

```diff
@@ -253,8 +288,9 @@ goog.require('ga_urlutils_service');
           group.append('text')
               .attr('class', 'ga-profile-label ga-profile-label-x')
               .attr('x', width / 2)
-              .attr('y', height + options.margin.bottom - 2)
+              .attr('y', height + 2 * options.margin.bottom / 3)
```

Comment:
The problem of this formula is if we change `options.margin.bottom` this couldn't work as expected . I prefer to put a magic number  `height + options.margin.bottom  - 30 `  or something like that.